tests/collections/collection.test.ts

```diff
@@ -116,7 +115,7 @@ for (const testClient in testClients) {
           error = e;          
         }
         assert.ok(error);
-        assert.strictEqual(error.errors[0].message, "insertMany can not take more than 100 docs");
+        assert.strictEqual(error.errors[0].message, "Request invalid, the field postCommand.command.documents not valid: amount of documents to insert is over the max limit.");
```

Comment:
This is probably not on client side, but error messages should ideally contain information on what the limit is, as well as how many were sent. I'll file an issue on json-api to see if we could improve this

Reply:
Filed https://github.com/stargate/jsonapi/issues/365 and will submit a PR adding missing information

Reply:
Thanks @tatu-at-datastax The new message from the JSON API will reflect in the stargate-mongoose side once the next version of JSON API is published and that gets updated here as well.